_posts/2019-11-20-bazel-1.2.0.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: posts
+title: "Bazel 1.2"
+authors:
+  - philwo
+---
+
+We've just released [Bazel 1.2](https://github.com/bazelbuild/bazel/releases/tag/1.1.0)!
+
+Bazel 1.2 is intended to be backward-compatible with Bazel 1.0, however it contains a single incompatible change that was deemed a bug fix and thus eligible for inclusion.
+
+Please [report](https://github.com/bazelbuild/bazel/issues/new) any update problems.
+
+## Incompatible changes
+
+* Tree artifacts and regular artifact paths can no longer overlap. This could happen when a Starlark action creates a tree artifact and a file artifact underneath the tree. This triggered undefined behavior (e.g. what should a downstream action that depends on the tree artifact see?) and is now explicitly forbidden.
+
+## Android
+
+* `android_binary` and `android_library` can now depend on targets providing `CcInfos`. See [#9079](https://github.com/bazelbuild/bazel/issues/9079) for details.
+* Experimental support for d8 dexers (both standalone and incremental) is now available for use using `--nouse_workers_with_dexbuilder` `--define=android_incremental_dexing_tool=d8_dexbuilder`. See [#5150](https://github.com/bazelbuild/bazel/issues/5150) for details.
+* Added Desugar support for `FreezePeriod#<init>`.
+## Query
+
+* Fixed an aquery bug with handling malformed queries that crashes Bazel.
+
+## C++
+
+* The `cc_common.{compile,link}` APIs can now be used without passing the `--experimental_cc_skylark_api_enabled_packages` flag.
+* Incompatible flag `[--incompatible_validate_top_level_header_inclusions](https://github.com/bazelbuild/bazel/issues/10047)` has been added.
```

Comment:
@laurentlb Is this the correct format to turn code-formatted words into links for our blog? Or do we have to put the backticks into the square brackets?

Reply:
I put backticks inside square brackets

Reply:
No! Put the \` inside the `[]`. Tags are not interpreted inside the backquotes.

Reply:
It sounds like I'm not the first one who did it like this ;) Fixed, thank you!